### DISCUSSION TITLE
Mac os application is not running

### DISCUSSION THREAD

**Discussion by khagenbaro** [eck4god/BibleApp#19] [Q&A]
Hi is there anyone who can help me to run the application in my mac. I wanna contribute. If anyone is there to guide me through the project I would be very grateful. Thanks

**Comment by GEchelberry** [comment]
I would love the help. I built the initial application on IntelliJ, Sqlite3 would also need to be installed.  Is there any specific errors that you are receiving?

**Comment by khagenbaro** [comment]
Please refer to the image.
I am new to sqllite3 . So I need help with setup of the project with
sqllite3.

Thanks

**Comment by GEchelberry** [comment]
To install Sqlite3 download from https://www.sqlite.org/index.html. I didn't see an image come through if you could try to repost it

**Comment by khagenbaro** [comment]
ok what to do after installing sqlite?

**Comment by GEchelberry** [comment]
Are you using IntelliJ as an IDE

**Comment by khagenbaro** [comment]
<img width="656" alt="bible_reader_error" src="https://github.com/eck4god/BibleApp/assets/74665369/6e3047f0-9287-4754-9728-e5bffb52f110">

yes i am using intellij ide

**Reply by GEchelberry** [reply]
Ah, that is a missing file in the Resources folder, Did you download the resource folder? If so make sure that it is in the BibleApp directory and config.json is listed

**Comment by khagenbaro** [comment]
<img width="756" alt="Screenshot 2023-08-09 at 11 57 59 PM" src="https://github.com/eck4god/BibleApp/assets/74665369/ca4cb492-870b-46d4-b247-77c58edd7c6e">
 this is the config file

**Comment by khagenbaro** [comment]
its showing file not found exception.

**Comment by GEchelberry** [comment]
It looks like it isn't finding the file in the data structure. I didn't think about slight changes in the file structure when it is installed on other systems. This is the first project I am making open source. If you're up to correcting this bug, changing how the program accesses files, to use the absolute path, it was something I started to think about. I saw a few ways to implement that on stack overflow. Once the files are found it should work

**Comment by khagenbaro** [comment]
yes . i given the full path it worked. but now its showing error as connecting to database
so please help me to connect with sqllite3

**Comment by GEchelberry** [comment]
in the DatabaseConnection class in the constructor, change the path in the String url

**Comment by khagenbaro** [comment]
ok

**Comment by GEchelberry** [comment]
Did you get it to work?

**Comment by khagenbaro** [comment]
Thanks its working perfectly

**Reply by GEchelberry** [reply]
Awesome, I create an issue if you come up with a more permeant fix.
